File: unity/Assets/Hallway/Constructor/RPG/terminalJump.js
#pragma strict

var Bootstrap : GameObject;

function OnTriggerEnter() {
	var scr1 : RPGParser = Bootstrap.GetComponent.<RPGParser>();
	if (!scr1.RPG.terminalJump) {
		scr1.Achievement("Прыжок на терминал!\n+10 очков!", 10);
		scr1.RPG.terminalJump = true;
	}
}

File: unity/Assets/Hallway/Constructor/RPG/terminalJump.cs
using UnityEngine;

public class terminalJump : MonoBehaviour
{
    public GameObject Bootstrap;

    void OnTriggerEnter()
    {
        RPGParser rpgParser = Bootstrap.GetComponent<RPGParser>();
        if (!rpgParser.RPG.terminalJump)
        {
            rpgParser.Achievement("Прыжок на терминал!\n+10 очков!", 10);
            rpgParser.RPG.terminalJump = true;
        }
    }
}
